packages/vector_graphics/lib/src/render_vector_graphics.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io';
```

Comment:
This breaks web support right?

Reply:
yeah - I could conditionally import it, we don't have any web users yet though

Reply:
done